### DISCUSSION TITLE
UMD and PageSpeed

### DISCUSSION THREAD

**Discussion by jamesli2021** [quasarframework/quasar#15431] [General - Components / Directives / etc]
I tried to experiment loading Quasar UMD over CDN and self-host on Github using 2.11.5.

The objective was to render a marketing site with Quasar, I get PageSpeed's Total Block Time for 250mw (mobile test). Whereas testing on Astro Build (https://astro.build/) with no JavaScript, could reach 100/100 Lighthouse score, 0 CLS, 0 TBT.

I like to explore it's feasible to bring down TBT to 0 or close to 0?

**Comment by jamesli2021** [comment]
When going SSR route, I tried to aim for 100/100 Lighthouse to find Quasar material UI need further optimization on accessibility and unused CSS.

**Comment by etewiah** [comment]
I would be interested too in know what optimisations people are doing to improve lighthouse speed for Quasar.  Or if it is possible to somehow combine quasar with astro.
